packages/dotcom-server-app-context/src/schema.json

```diff
@@ -6,49 +6,74 @@
     "abTestState": {
       "type": "string",
       "description": "The A/B test flags data as a comma delimited string",
-      "examples": ["subscriberCohort:on,premiumCohort:on,nonUSACohort:on"],
+      "examples": [
+        "subscriberCohort:on,premiumCohort:on,nonUSACohort:on"
+      ],
       "pattern": "^,*([0-9A-Za-z-_]+:[0-9A-Za-z-_]+,*)+$"
     },
     "appName": {
       "type": "string",
       "description": "The name of the application",
-      "examples": ["front-page", "stream-page", "article-page"],
+      "examples": [
+        "front-page",
+        "stream-page",
+        "article-page"
+      ],
       "pattern": "^.+$"
     },
     "appVersion": {
       "type": "string",
       "description": "The running version of the app (usually a Git commit hash)",
-      "examples": ["882797258625531f20d604f6441ef8cfcb2d772b"],
+      "examples": [
+        "882797258625531f20d604f6441ef8cfcb2d772b"
+      ],
       "pattern": "^.+$"
     },
     "conceptId": {
       "type": "string",
       "description": "The UUID of the concept on the current page",
-      "examples": ["c5935758-7730-11e9-bbad-7c18c0ea0201"],
+      "examples": [
+        "c5935758-7730-11e9-bbad-7c18c0ea0201"
+      ],
       "pattern": "^[0-9a-z]{8}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{12}$"
     },
     "conceptType": {
       "type": "string",
       "description": "The type of concept on the current page",
-      "examples": ["http://www.ft.com/ontology/product/Brand", "http://www.ft.com/ontology/Location"],
+      "examples": [
+        "http://www.ft.com/ontology/product/Brand",
+        "http://www.ft.com/ontology/Location"
+      ],
       "pattern": "^http://www.ft.com/ontology/.+$"
     },
     "contentId": {
       "type": "string",
       "description": "The UUID of the content on the current page",
-      "examples": ["c5935758-7730-11e9-bbad-7c18c0ea0201"],
+      "examples": [
+        "c5935758-7730-11e9-bbad-7c18c0ea0201"
+      ],
       "pattern": "^[0-9a-z]{8}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{12}$"
     },
     "contentType": {
       "type": "string",
       "description": "The type or sub-type of the content on the current page",
-      "examples": ["article", "video", "audio", "podcast", "package", "live-blog"],
-      "pattern": "^(article|video|audio|podcast|package|live-blog)$"
+      "examples": [
+        "article",
+        "video",
+        "audio",
+        "podcast",
+        "package",
+        "live-blog"
+      ],
+      "pattern": "^(Article|Placeholder|Video|Audio|LiveBlogPackage|LiveBlogPost|ContentPackage|Content|MediaResource)$"
```

Comment:
**thought**: it occurs to me that this implicitly ties Page Kit to cp-content-pipeline, and if we ever add another content type, we'd need to update this, and it won't be obvious that we need to. maybe we should make the schema more lenient.